### PR TITLE
Add docs for PurpleAir integration

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/82110
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3857
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/purpleair.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: PurpleAir
+description: Instructions on how to integrate PurpleAir sensors with Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 2022.12.0
```
Comment (frenck) [inline source/_integrations/purpleair.markdown]:
```suggestion
ha_release: 2023.1.0
```

File: source/_integrations/purpleair.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ha_codeowners:
+  - '@bachya'
+ha_domain: purpleair
+ha_platforms:
+  - sensor
+ha_integration_type: device
+ha_config_flow: true
+ha_integration_type: device
```
Comment (frenck) [inline source/_integrations/purpleair.markdown]: Duplicate

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by bachya** [comment]
Appreciated, @frenck!

File: source/_integrations/purpleair.markdown [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: PurpleAir
+description: Instructions on how to integrate PurpleAir sensors with Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 2023.1.0
```
Comment (frenck) [inline source/_integrations/purpleair.markdown]:
```suggestion
ha_release: 2023.1
```

**Review by frenck — APPROVED** [review]
Thanks, @bachya 👍

File: source/_integrations/purpleair.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/purpleair.markdown]: My bad, should have noticed. Sorry!

**Pull request merged**